test/cusolver/cusolvermg.jl

```diff
@@ -0,0 +1,114 @@
+using LinearAlgebra, Test
+using CUDA
+using CUDA.CUSOLVER
+
+import LinearAlgebra: BlasInt
+
+@testset "CUSOLVERMG" begin
+
+m = 256
+n = 512
+devs = collect(devices())
+
+if CUDA.has_cusolvermg()
```

Comment:
similarly, this check should be moved to runtests.jl, where we also test for e.g. cudnn